<launch>
  <node pkg="lidar_3d_roi_pkg" type="lidar_3d_roi_node" name="lidar_3d_roi_node" output="screen">
    <param name="x_min" value="0.0" />
    <param name="x_max" value="100.0" />
    <param name="y_min" value="-40.0" />
    <param name="y_max" value="40.0" />
    <param name="z_min" value="-0.3" />
    <param name="z_max" value="15.0" />
  </node>
</launch>
